docs/can-guides/contribute/including-new-modules.md

```diff
@@ -0,0 +1,111 @@
+@page guides/contributing/include-new-modules Including New Modules
```

Comment:
Maybe “Adding Packages to CanJS.com”? I’m weary of putting “ecosystem” in the name because it applies to all of the collections.